packages/common/src/search/wellKnownCatalog.ts

```diff
@@ -0,0 +1,266 @@
+import { IUser } from "@esri/arcgis-rest-types";
+import { getFamilyTypes } from "../content";
+import { EntityType, IHubCatalog, IHubCollection } from "./types";
+
+export type WellKnownCatalog =
```

Comment:
Please add JSDoc Comments describing the reason for this type and where it's used. Same for `WellKnownCollection`

Reply:
typo - `determine` :)